Source/WebCore/dom/Document.h

```diff
@@ -1844,6 +1846,12 @@ class Document
 
     void detachFromFrame();
 
+    Ref<ViewTransition> startViewTransition(RefPtr<ViewTransitionUpdateCallback> = nullptr);
```

Comment:
These might be better moved up in the public section, if there's some logical grouping with other functions. Don't always just add stuff at the end.